autoload/ale/fixers/format.vim

```diff
@@ -0,0 +1,25 @@
+" Author: soywod <clement.douin@gmail.com>
+" Description: Integration of Elm with ALE.
+
+call ale#Set('elm_format_executable', 'elm-format')
+call ale#Set('elm_format_use_global', 1)
```

Comment:
Set this to `0` by default, to match the other options. Local versions should be tried first, then global versions, by default.

Reply:
Done